torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -456,11 +560,23 @@ void ComputeSubgraphInMKLDNN(Node* subgraph_node) {
       body_node->replaceInput(1, node->outputs().at(1));
     }
 
-    if (body_node->kind() == aten::conv2d ||
-        body_node->kind() == aten::conv3d) {
-      // this node doesnt handle string padding yet...
-      if (!body_node->namedInput("padding")->type()->cast<StringType>()) {
-        body_node->replaceWithNewSymbol(Symbol::prim("mkldnn_convolution"));
+    auto true_pred = [](Node*) { return true; };
```

Comment:
Nit: can you keep the conv handling how it was originally? I think it was more readable originally and we've lost the `this node doesnt handle string padding yet` comment. If we start getting a lot more predicates than maybe we should refactor.